.mergify.yml

```diff
@@ -1,9 +1,11 @@
 pull_request_rules:
-  - name: backport to master
+  # For FireSim version 1.x.y, here let x = minor, y = patch
+  # Only support backporting to the last minor release branch.
+  # This rule will need to be updated on minor releases.
+  - name: backport to latest minor release
     conditions:
       - merged
       - base=main
-      - milestone=1.13.x
```

Comment:
This seems like a reasonable change to me.  It's annoying that you have to hardcode the target branch and update that on every release below...

Reply:
This does seem like an easy thing to automate though. I assume there is some GH-A that can do some regex-ing and replace the string based on the release tag...

Reply:
I'll track more release automation features in a follow up issue.